main.py

```diff
@@ -88,6 +89,7 @@ def limit_memory():
     command = [
         str(libjs_test262_runner),
         *(["-b"] if use_bytecode else []),
+        *(["-e"] if enable_bytecode_optimizations else []),
```

Comment:
Isn't this supposed to be `-p`? :thinking:

Reply:
nope, it's `-p` in `js`, but for `test262-runner` `p` is already used for `parse-only`, so in my matching serenity PR I added `e` for optimizations.

Reply:
ah, gotcha! On a related note, one-letter options were a mistake and never should've been invented. :upside_down_face: